### ISSUE TITLE
Chrome MediaStreamTrack.getSources is deprecated

### ISSUE THREAD

**Issue by karth295** [meetecho/janus-gateway#499]
I get this deprecation warning in the console using Chrome Version 49.0.2623.110 (64-bit) on a MacBook:

```
MediaStreamTrack.getSources is deprecated. See https://www.chromestatus.com/feature/4765305641369600 for more details. (janus.js:1486)
```

Instead, they recommend using MediaDevices.enumerateDevices(), which is supported in both chrome and firefox: https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/enumerateDevices

**Comment by lminiero** [comment]
Already addressed in #476, which is waiting for feedback before I can merge it. If you care about that, please test that branch (and the new demo it adds) and let us know if that improves things.

**Comment by lminiero** [comment]
Merged that PR in the meanwhile, so closing the issue here.
